README.md

```diff
@@ -22,8 +22,8 @@ configuration changes, updates, upgrades, and rollbacks.
 
 ## Prerequisites
 
-- Kubernetes 1.10+
-- Helm 2.13+
+- Kubernetes 1.13+
+- Helm 2.10+
```

Comment:
Can you check if Helm 2.10 has the `--set-file` option? It looks like it was added with 2.10 (https://github.com/helm/helm/pull/3758) but just to make sure. We use that option in the tutorial.

Reply:
Yes it does.